### DISCUSSION TITLE
Logbook infos - Source of action

### DISCUSSION THREAD

**Discussion by smileyman-1** [make-all/tuya-local#2061] [Q&A] [answered]
For some events, the logbook records who triggered them (e.g. automation by changing an entity value). For other events, however, it does not.
Both marked actions were triggered by the same automation. However, when opening, it always says that it was the automation. When closing, however, it does not. (I think I have seen this before, though.)
How do I make this generally available to Infor?

<img width="852" alt="image" src="https://github.com/make-all/tuya-local/assets/55857747/370765e1-910f-4e93-a207-b8e046cc60b7">

Thanks for a advice

**Comment by make-all** [comment, marked as answer]
This is a general Home Assistant question, you might be better asking in the Home Assistant forums or Discord.

As a backend integration, tuya-local does not know who/what triggered an action.

**Comment by smileyman-1** [comment]
Thanks for your answer - I thought it was due to the Tuya integration.
Then I'll ask in the HA community
